packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/apple/appatidp.md

```diff
@@ -0,0 +1,65 @@
+
+There are three steps necessary to configure the Identity Provider at Apple:
+
+* **Create an App ID** &mdash; An App ID enables your app to access available services and identifies your app in a provisioning profile with Apple.
+* **Create a Services ID** &mdash; The Services ID identifies the particular instance of your app. The Services ID is used as the OAuth `client_id`.
+* **Create a private key for client authentication** &mdash; Apple uses a public/private key pair as the client secret. You must register a new private key with Apple and download the key file for use with Okta.
+
+## Create an App ID
+
+1. From the [Apple Developer Dashboard](https://developer.apple.com/) left navigation, click **Certificates, IDs, & Profiles**.
+
+2. Select **Identifiers** from the left and then click the blue plus icon.
+
+3. Select **App IDs** if it isn't already selected, and then click **Continue**.
+
+4. Enter a description and a Bundle ID for the App ID. Apple recommends using a reverse-domain name style string for the Bundle ID (for example, if the domain your app runs on is `example.com`, then enter `com.example` as the Bundle ID.)
+
+    > **Note:** You can't edit this identifier after you create it, you can only remove it and create a new one.
+
+5. Scroll down and select the **Sign In with Apple** check box, click **Continue**, and then **Register**.
+
+## Create a Services ID
+
+1. Create another identifier, this time selecting **Services IDs** from the drop-down box on the right.
+
+2. After you click **Continue**, enter a name for the identifier in the **Description** box. This is what the user sees when they sign in.
+
+3. In the **Identifier** box, enter the identifier in a reverse-domain name style string (for example, if the domain your app runs on is `example.com`, then enter `com.example.client` as the identifier.). This identifier becomes the OAuth `client_id`.
+
+    > **Note:** The **Services ID** identifier can't be the same as the **App ID** identifier.
+
+4. Click **Continue** and then **Register**. The **Identifiers** page appears.
+
+5. Select the identifier that you just created.
+
+6. Scroll down and select the **Sign In with Apple** check box, and then click **Configure** to define the domain your app is running on and the redirect URLs used during the OAuth flow.
+
+7. If it isn't already selected, select the App ID that you just created from the **Primary App ID** drop-down box.
+
+8. Enter the domain name where your app runs in the **Domains and Subdomains** box. Be sure to enter a real domain. Domains such as `localhost` or IP addresses such as `127.0.0.1` won't work.
+
+9. In the **Return URLs** box, enter a placeholder redirect URL such as `https://example-app.com/redirect`. We come back and add the correct redirect URL later. The redirect URL is where you want to redirect users after successfully signing in to your website.
```

Comment:
This step can be simplified. The callback URL never changes on Okta side. So we could say that your callback URL will be <your org name>/oauth2/v1/authorize/callback. So if your Okta subdomain is company then the callback value is https://company.okta.com/oauth2/v1/authorize/callback. If you have configured custom domain on the Okta Org use that value to construct your redirect URI